### ISSUE TITLE
Mailvelope does not respond in Kopano Webapp

### ISSUE THREAD

**Issue by davodego** [mailvelope/mailvelope#707]
The option to encrypt is not displayed in the Kopano Webapp dialogs.

Browser Vivaldi 2.8.1664.40 (Stable channel) (64-bit)

In Firefox it works with Mailvelope and Kopano Webapp dialogs.

Can you fix this?

Edit 02.11.2019
Write a encrypted message woks. Mailvelope noticed the editor an I can open the window to write a secure mail.
When the encrypted mail arrived in Kopano WebApp, the Mailvelope Icon was not displayed to decrypt the message.

**Comment by fbartels** [comment]
fyi: there is also a really old issue for the old Zarafa WebApp already https://github.com/mailvelope/mailvelope/issues/27

**Comment by toberndo** [comment]
This should be fixed with https://github.com/mailvelope/mailvelope/commit/37cb020a21f321b5f15b6a92cbb6d48e140c6295 as we scan now also in iframes for PGP message.
Can be tested with latest builds on https://build.mailvelope.com/nightly/

**Comment by fbartels** [comment]
Hi @toberndo,

I have installed the version from `mailvelope.2020-05-23.chrome` and could encrypt an outgoing message (to myself). When now clicking on the encrypted mail the password window seemingly gotten in the background and I was wondering for a bit why I did not ask for a password when reloading the mail or decrypting the message text directly in the extension. And when starting to compose this message to you I finally found it again 😅.

From a quick test it seems to work great. I also like the redesign of Mailvelope, its been a few years since I have used it.

I think this issue is resolved with 4.3.2 and can be closed.
